src/arch/host/include/arch/string.h

```diff
@@ -32,7 +32,53 @@
 #ifndef __INCLUDE_ARCH_STRING_SOF__
 #define __INCLUDE_ARCH_STRING_SOF__
 
+#include <errno.h>
+
 #define arch_memcpy(dest, src, size) \
 	memcpy(dest, src, size)
 
+#define arch_memset(ptr, size) \
```

Comment:
`arch_memset()` should take another 'ch' argument

Reply:
It is used for bzero operation on buffers - will change name to reflect that better.